Profile/Profile/SwiftGen/Strings.swift

```diff
@@ -14,8 +14,14 @@ public enum ProfileLocalization {
   public static let bio = ProfileLocalization.tr("Localizable", "BIO", fallback: "Bio:")
   /// Contact support
   public static let contact = ProfileLocalization.tr("Localizable", "CONTACT", fallback: "Contact support")
+  /// Cookie Policy
+  public static let cookiePolicy = ProfileLocalization.tr("Localizable", "COOKIE_POLICY", fallback: "Cookie Policy")
+  /// Do Not Sell my Personal Information
+  public static let doNotSellInformation = ProfileLocalization.tr("Localizable", "DO_NOT_SELL_INFORMATION", fallback: "Do Not Sell my Personal Information")
```

Comment:
I'm not sure but you might need to follow the screen pattern for labels. So you need to use only the first character in caps in the whole text.

Reply:
@saeedbashir  we are trying to follow current app as mentioned here https://github.com/openedx/openedx-app-ios/issues/117
Do we need to change this string @moiz994 ?
Thank you

Reply:
@rnr we have discussed and I changed